### PR TITLE
Add changes for version 23.5.0

### PR DESCRIPTION
**Pull request by marcoesters**
Add release notes, bug fixes, and known issues to release notes for Miniconda `23.5.0`

### CODE REVIEW

File: docs/source/miniconda_releases/23.5.0-1/changes.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+User-facing changes
+^^^^^^^^^^^^^^^^^^^
```
Comment (Jrice1317) [inline docs/source/miniconda_releases/23.5.0-1/changes.rst:2]:
```suggestion
-------------------
```

File: docs/source/miniconda_releases/23.5.0-1/changes.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+* Conda has been updated to v23.5.0
+* OpenSSL has been updated to version 3
+* The default installer uses python-3.11.3
+
+
+Bug fixes
+^^^^^^^^^
```
Comment (Jrice1317) [inline docs/source/miniconda_releases/23.5.0-1/changes.rst:10]:
```suggestion
---------
```

File: docs/source/miniconda_releases/23.5.0-1/changes.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+* Address a bug that can cause accidental file deletion with Windows uninstallers (see our `blog post <https://www.anaconda.com/blog/windows-installer-security-fix>`_). A `security patch <https://repo.anaconda.com/miniconda/Miniconda3-uninstaller-patch-win-64-2023.07-0.exe>`_ is available for older versions.
+
+
+Known issues
+^^^^^^^^^^^^
```
Comment (Jrice1317) [inline docs/source/miniconda_releases/23.5.0-1/changes.rst:16]:
```suggestion
------------
```

File: docs/source/miniconda_releases/23.5.0-1/changes.rst [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+* The .pkg installers may skip the "Destination Select" page after accepting the license agreement (see our `Anaconda installation instructions<https://docs.anaconda.com/free/anaconda/install/mac-os/>`_ for details)
```
Comment (Jrice1317) [inline docs/source/miniconda_releases/23.5.0-1/changes.rst]:
```suggestion
* The .pkg installers may skip the "Destination Select" page after accepting the license agreement. See our `Anaconda installation instructions <https://docs.anaconda.com/free/anaconda/install/mac-os/>`_ for details.
```

**Review by Jrice1317 — CHANGES_REQUESTED** [review]

File: docs/source/miniconda_releases/23.5.0-1/changes.rst (hunk 1 above)
Comment (marcoesters) [inline docs/source/miniconda_releases/23.5.0-1/changes.rst:2]: This is in line with the Anaconda release notes. See the following:

```
=============
Release notes
=============

Anaconda 2023.03-1 (April 24, 2023)
-----------------------------------

User-facing changes
^^^^^^^^^^^^^^^^^^^

* Conda has been updated to v23.3.1.

```

Why is Miniconda different?

Comment (Jrice1317) [reply docs/source/miniconda_releases/23.5.0-1/changes.rst:2]: Okay, just hold off on my PR, I will make some changes.

**Review by Jrice1317 — APPROVED** [review]
Because our template title uses equal signs, we must use dashes for our second-level

**Pull request merged**